src/lang/KclSingleton.ts

```diff
@@ -603,8 +606,11 @@ export class KclManager {
     return Promise.all(thePromises)
   }
   /** TODO: this function is hiding unawaited asynchronous work */
-  defaultSelectionFilter() {
-    setSelectionFilterToDefault(this.programMemory, this.engineCommandManager)
+  defaultSelectionFilter(restorePreviousSelections?: Selections) {
```

Comment:
[nit] `restorePreviousSelections` sounds like a boolean to me, could we consider `selectionsToRestore`?

Reply:
Done.